### DISCUSSION TITLE
What's the best practice for LXC Backup

### DISCUSSION THREAD

**Discussion by mkroemer** [community-scripts/ProxmoxVE#609] [Q&A]
I would like to know what's the best practice to backup the LXCs created with the community scripts. Doing a full backup through proxmox or just the /etc, /var/lib and /opt directories and copy them into a newly created LXC if necessary.

**Comment by knario** [comment]
Hi, I have it through proxmox backup server on a vm. And also on tuxis. They give you about 150Gb free for backups.
